shared/util/quit-helper.desktop.js

```diff
@@ -0,0 +1,86 @@
+// @flow
+import electron from 'electron'
+import {ipcRenderer, ipcMain} from 'electron'
+import {quit} from '../../desktop/app/ctl'
+
+const app = electron.app || electron.remote.app
```

Comment:
Also then we don't have to do all these weird checks for what process we are on?